BUILD.BAT

```diff
@@ -9,17 +9,18 @@ if /I "%1" == "tron" (set z_TraceMode=tron
                                      ) else (set z_TraceMode=)
                       )
 set /A z_NestLevel=%z_NestLevel%+1
-rem ----- Lvl(%z_NestLevel%) Start %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
 pushd %~dps0
```

Comment:
Please put removed rem lines (lines 12 and 31 in original version) back in. These are extremely useful when debugging bat files in trace mode! They log entry into and exit from each bat file with the exact parameter values passed on entry.